### ISSUE TITLE
Crash when loading IBM advert on DDG

### ISSUE THREAD

**Issue by jonathanKingston** [mozilla-mobile/focus-android#464]
I was getting a consistent crash doing the following STR:

- "Network manager ubuntu" using DuckDuckGo
  https://duckduckgo.com/?q=network+manager+ubuntu&t=fpas&ia=web
- Clicking the following first advert:
  http://www-935.ibm.com/services/us/en/it-services/networking-services/?S_PKG=-&cm_mmc=Search_Bing-_-GTS_GTS+Networking+Services-_-WW_IUK-_-+networking++management_Broad_-&cm_mmca1=000000TJ&cm_mmca2=10000687&mkwid=e1ca756e-9e5d-4c57-8ac5-f79b5aec4b80|449|295733&cvosrc=ppc.bing.%2Bnetworking%20%2Bmanagement&cvo_campaign=GTS_GTS%20Networking%20Services-WW_IUK&cvo_crid=15867273821&Matchtype=p

I have the following blocked:
- add trackers
- analytics trackers
- social trackers
- 'other content trackers'

Searching "935 ibm network services" also gave me the same advert.

I'm using a Google Pixel in the UK.

**Comment by pocmo** [comment]
Looks like this might be:
https://dashboard.buddybuild.com/apps/584f67f6f3d6eb01000842d6/crashreport/0bc41cbd3662d94034bf0c98f3e9d4b8#threads

```
java.lang.IllegalStateException: WebView is expected to always supply a valid URL
0	org​.mozilla​.focus​.utils	UrlUtils.java ​Line ​81
UrlUtils.stripUserInfo
	
1	org​.mozilla​.focus​.fragment	BrowserFragment.java ​Line ​81
BrowserFragment.updateURL
	
2	org​.mozilla​.focus​.fragment	BrowserFragment.java ​Line ​36
BrowserFragment.access$500
	
3	org​.mozilla​.focus​.fragment	BrowserFragment.java ​Line ​185
BrowserFragment$1.onURLChanged
	
4	org​.mozilla​.focus​.web	WebViewProvider.java ​Line ​275
WebViewProvider$WebkitView$2.onProgressChanged
```

**Comment by jonathanKingston** [comment]
@pocmo I can't see that crash report so not sure, there should be 5ish of them assuming it sends it each time.

Is there anyway for me to check it is this?

**Comment by pocmo** [comment]
I don't think there's a way to make them public in buddybuild. However I can see that they all came from buddybuild tester "jkingston" - so I guess that's you. :)

**Comment by jonathanKingston** [comment]
It is indeed. Thanks for checking :)

**Comment by ahunt** [comment]
So this is already "fixed", in #414:
3bd81266d40bc3ae9306c0518a263aa3fe4dae0a

That landed in Build 1082 (your crashes are build 1064 - and that's the last build we pushed out to the larger group of testers). I'll try to get an updated build pushed out!
